packages/protocol/src/vaults/borrowing/BorrowingVault.sol

```diff
@@ -544,7 +515,8 @@ contract BorrowingVault is BaseVault {
     uint256 debtShares = _debtShares[borrower];
     uint256 debt = convertToDebt(debtShares);
 
-    uint256 baseUserMaxBorrow = ((assets * maxLtv * price) / (1e18 * 10 ** decimals()));
+    // uint256 baseUserMaxBorrow = ((assets * maxLtv * price) / (1e18 * 10 ** decimals()));
+    uint256 baseUserMaxBorrow = assets.mulDiv(maxLtv.mulDiv(price, 1e18), 10 ** decimals());
```

Comment:
I think the math is a bit different now, there could be precision loss. This is the new formula, based on the new code
```solidity
assets*(maxLtv*price/1e18)/10**decimals()
```
doing instead
```solidity
assets.mulDiv(maxLtv*price, 10 ** decimals()*1e18);
```
would prevent precision loss
the risk would be `maxLtv*price` overflow